docs/rules/no-anonymous-default-export.md

```diff
@@ -0,0 +1,51 @@
+# no-anonymous-default-export
+
+Reports if a module's default export is unnamed. This includes several types of unnamed data types; literals, object expressions, anonymous functions, arrow functions, and anonymous class declarations.
```

Comment:
Agreed - it should handle arrays, but template literals i'd expect to be covered under "literals".